exec/cnex/lib/time_win32.c

```diff
@@ -0,0 +1,48 @@
+#include <iso646.h>
+#include <stdlib.h>
+
+#include "cell.h"
+#include "exec.h"
+#include "number.h"
+#include "stack.h"
+#include "time.h"
+
+#include <windows.h>
+
+
+static const ULONGLONG FILETIME_UNIX_EPOCH = 116444736000000000ULL;
+static Number PERFORMANCE_FREQUENCY;
+
+void time_initModule()
+{
+    LARGE_INTEGER freq;
+    QueryPerformanceFrequency(&freq);
+    PERFORMANCE_FREQUENCY = number_from_uint64(freq.QuadPart);
+}
+
+void time_sleep(TExecutor *exec)
+{
+    Number seconds = top(exec->stack)->number; pop(exec->stack);
+
+    uint32_t ms = number_to_uint32(number_multiply(seconds, number_from_uint32(1000)));
```

Comment:
What happens if this overflows a uint32_t?

Reply:
Nothing?  It would have to sleep for 24 days for that to be an issue.  None of the other time functions have any exception handling around them, so I assumed that since `Sleep()` takes a uint342_t, I'd just declare it as such.